cmd/planner-agent/main.go

```diff
@@ -77,9 +76,9 @@ func (a *agentCmd) Execute() error {
 	undo := zap.ReplaceGlobals(logger)
 	defer undo()
 
-	agentID, err := a.readFile(agentFilename)
+	agentID, err := a.readPersistentFile(agentFilename)
```

Comment:
why change of name here? what value `Persistent` bring here?

Reply:
why not adding changing the initial method and just add a parameter with the folder, or better just path the whole path and do the join here

Reply:
There is some code duplication here indeed, let me think how to refactor - obviously a common underlying function with a param for basedir, but I still want two separate names to call it, so that I do not need to think about the right config option, just know if I am reading from the persistent storage or the volatile one,